app/controllers/member_conversion_controller.rb

```diff
@@ -0,0 +1,202 @@
+class MemberConversionController < ApplicationController
+  include CurationConcerns::Lockable
+
+  #Promote a FileSet to a child GenericWork.
+  def to_child_work
+    begin
+      parent_work = GenericWork.find(params['parentid'])
+      file_set = FileSet.find(params['filesetid'])
+    rescue
```

Comment:
No bare `rescue` please, rescue the proper class. I think in this case some kind of `NotFound` exception, not sure which one?

Reply:
Oops - thanks for catching that.